app/controllers/signup.js

```diff
@@ -2,10 +2,13 @@ import Controller from '@ember/controller';
 import { action, set } from '@ember/object';
 import { tracked } from '@glimmer/tracking';
 import registerUser from '../utils/register-api';
+import mixpanel from 'mixpanel-browser';
 import ENV from 'website-my/config/environment'; // remove this when new flow goes live
 
 const BASE_URL = ENV.BASE_API_URL; // remove this when new flow goes live
 
+mixpanel.init('5fcfb02eabfc77f5a6a0e4cb65bbf5e0');
```

Comment:
Let us please have it becoming from environment variables? This would leak our key to the public This should help: https://stackoverflow.com/questions/26403334/how-to-pass-api-keys-in-environment-variables-to-ember-cli-using-process-env

Reply:
Made the change, Thank you @swarajpure I was missing something previously I guess that's why there were errors